hedera-mirror-rest/api/v1/openapi.yml

```
@@ -3293,3 +3293,17 @@ components:
       schema:
         type: boolean
         default: false
+    transactionIdOrEthHashPathParam:
+      name: transactionIdOrHash
+      in: path
+      required: true
+      description: Transaction Id or a 32 byte hash with optional 0x prefix
+      examples: 
+        trahsactionId:
+          value: 0.0.10-1234567890-000000000
+        transactionHash:        
+          value: ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef
+        transactionHashWith0xPrefix:        
+          value: '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'
+      schema:
+        type: string
```

add a schema for transaction id string if there isn't one, then change this to
```suggestion
      schema:
        oneOf:
          - $ref: '#/components/schemas/EthereumHash'
          - $ref: '#/components/schemas/TransactionIdStr'
```

---

One exists already `$ref: '#/components/schemas/TransactionId'`

---

That's of type object not the string format we need here